pre-aio-node-with-zuul-ref.sh

```diff
@@ -0,0 +1,157 @@
+#!/bin/bash
+
+set -ex
+refspec="$1"
+ZUUL_CHANGES="$2"
+keep_env="$3"
+
+FETCH_URL=http://10.71.212.50/p/
+ROOT_DIR=/opt/stack/
+DEVSTACK_TOOL=/opt/stack/devstack
+NOVA_DIR=/opt/stack/nova
+TEMPEST_DIR=/opt/stack/tempest
+TMP_MERGE_BRANCH=tmp_branch_for_vgpu_patches
```

Comment:
I'd suggest use:
   TEST_BRANCH=$ZUUL_CHANGE

Reply:
Will change it